eclipse/tests/org.switchyard.tools.ui.tests/src/org/switchyard/tools/ui/tests/CreateSwitchYardProjectTest.java

```diff
@@ -353,4 +358,14 @@ protected void execute(IProgressMonitor monitor) throws CoreException {
         
     }
     
+    public void testVersionsListHasDefaultAndPreviousVersions() throws Exception {    
+        Set<ArtifactVersion> versions = new LinkedHashSet<ArtifactVersion>();
+        versions = M2EUtils.ensureDefaultAndPreviousVersionAvailable(versions);
+
+        final ArtifactVersion defaultVersion = M2EUtils.parseVersion(NewSwitchYardProjectWizard.DEFAULT_RUNTIME_VERSION);
+        final ArtifactVersion previousVersion = M2EUtils.parseVersion(NewSwitchYardProjectWizard.PREVIOUS_RUNTIME_VERSION);
+        
+        assertTrue("Versions list missing default SwitchYard Version", versions.contains(defaultVersion));
```

Comment:
I'm ok with the older approach, but I see your point. I'll look at more prescriptive assert statements as we go forward but don't see this as necessary for now.